src/components/BookList.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Newbook from './Newbook';
+
+const BookList = () => (
+  <div className="booklist">
+    Book List
```

Comment:
- [ Required ] I think it would be better if the BookList component actually renders a list of books using the `Book` component for now the data can come from an array of book objects. Please see the example below for more details on this, please feel free to improve it in any way that could satisfy the project requirements.

## Example code
```JavaScript
import React from 'react';
import Book from './Book';
import Newbook from './Newbook';

const BookList = () => {
  const books = [
    { title: 'Book I', author: 'Author I' },
    { title: 'Book II', author: 'Author II' },
    { title: 'Book III', author: 'Author III' },
  ];
  return (
    <div className="booklist">
      <ul>
        { books.map((book, index) => <Book key={index} title={book.title} author={book.author} />) }
      </ul>
      <Newbook />
    </div>
  );
};

export default BookList;

```